pkg/azdiskschedulerextender/azdiskschedulerextender_metrics.go

```diff
@@ -50,7 +50,7 @@ var (
 			Buckets:        metrics.ExponentialBuckets(0.0001, 2, 12),
```

Comment:
Given the latencies we typically see are <20us, I wonder if we should adjust the buckets to have better granularity at these times, i.e. start at 0.00001.